service-tests/liberapay.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'liberapay', title: 'Liberapay' });
+module.exports = t;
+
+t.create('Receiving')
+  .get('/Liberapay/receives.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'receives',
+    value: Joi.string().regex(/^[0-9]+(\.[0-9]{2})?[ A-Za-z]{4}\/week/)
+  }));
+
+t.create('Giving')
+  .get('/Liberapay/gives.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'gives',
+    value: Joi.string().regex(/^[0-9]+(\.[0-9]{2})?[ A-Za-z]{4}\/week/)
+  }));
+
+t.create('Patrons')
+  .get('/Liberapay/patrons.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'patrons',
+    value: Joi.string().regex(/^[0-9]+/)
```

Comment:
It's better to use `isMetric`, which will match all the variants and reject 0.

Reply:
It will reject value == 0? What if 0 is correct?

Reply:
Let's find a project that we're pretty confident will always have patrons.